django/core/handlers/base.py

```diff
@@ -179,8 +179,11 @@ def _get_response(self, request):
         """
         response = None
         callback, callback_args, callback_kwargs = self.resolve_request(request)
+        # Mark the request with sensitive_post_parameters if applied.
+        if hasattr(callback, "sensitive_post_parameters"):
+            request.sensitive_post_parameters = callback.sensitive_post_parameters
```

Comment:
Reading the ticket, it sounds like that this is called before the middleware methods is quite important.
Currently if I move this below where middleware is applied, the test still passes.
Can you add something in the test to cover that the request has sensitive_post_parameters set also in the middleware method?